### PR TITLE
#1 Added login page resolves #1

### LINKED ISSUE #1: Create "Login" Page
### Description
Create a basic login page based on [Figma](https://www.figma.com/file/MNRQAXLXzBqCR3aRZmXg0T/Habitat-for-Humanity). Include error handling for input fields. If an error occurs, an alert pops up describing the error. Store all input fields in a state variable so it can be formatted as a JSON object later

### Setup
1. Navigate to `/frontend/src/components/`
2. Create a folder with the [PascalCase](https://techterms.com/definition/pascalcase) naming convention (name based on the component you're working on)
3. Inside the folder, create a `.tsx` file with the same name as the folder
4. Be sure to update the `App.js` file with your new component, or else you wont see any changes you make. This file is located in `/frontend/src/`

### Success Metrics Checklist
- [x] Looks exactly like the Figma
- [x] Responsive screen handling
- [x] Error pops up for invalid/empty input field
- [x] Store submitted input into state variable

Finally, on your pull request, please request a code review from me (@ObjectJosh)

Screen Shot 2022-01-06 at 1 09 52 PM

### PR DESCRIPTION
**Pull request by tessa-tapmongkol**

### CODE REVIEW

File: frontend/src/components/LoginPage/LoginPage.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
+#loginBox {
+    height: 450px;
+    width: 350px;
+    background: #FEFEFE;
+    border: 0.5px solid #C4C4C4;
```
Comment (ObjectJosh) [inline frontend/src/components/LoginPage/LoginPage.css:5]:
```suggestion
    border: none;
```
I know the Figma shows a 0.5px border, but it doesn't show up in the app as expected, so I just got rid of it.

File: frontend/src/components/LoginPage/LoginPage.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    margin: auto;
+    margin-top: 18vh;
```
Comment (ObjectJosh) [inline frontend/src/components/LoginPage/LoginPage.css:7]:
```suggestion
    margin-top: 50vh;
    transform: translateY(-50%);
```
Centering the box on the screen

File: frontend/src/components/LoginPage/LoginPage.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+#loginForgotPassword {
+    color: #6DB340;
+    margin: 0;
+    font-weight: 500;
+    font-size: 13px;
+}
+
+.loginForgotPassword:hover {
```
Comment (ObjectJosh) [inline frontend/src/components/LoginPage/LoginPage.css:51]:
```suggestion
#loginForgotPassword:hover {
```
Just a small typo that caused this hover feature to not work

**Review by ObjectJosh — CHANGES_REQUESTED** [review]
Awesome work, Tessa! Just a few very minor changes, and you're set.

**Review by ObjectJosh — APPROVED** [review]

**Review by ObjectJosh — CHANGES_REQUESTED** [review]
One more thing -- it looks like you're removing a lot of lines from frontend/package-lock.json try going into /frontend then `npm i` just in case of any merge conflicts in the future. Let me know if you run into any issues.

**Comment by tessa-tapmongkol** [comment]
@ObjectJosh Hm I'm not sure why it says that, I did "npm i" but nothing changed

**Comment by ObjectJosh** [comment]
@tessa-tapmongkol Dang that's interesting. I tried it on my end from your branch and it does add all those lines back. I'm going to accept the merge. Hope all goes well

**Review by ObjectJosh — APPROVED** [review]

**Pull request merged**